onfido-java/src/main/java/com/onfido/Onfido.java

```diff
@@ -0,0 +1,44 @@
+package com.onfido;
+
+import com.onfido.api.Resources;
+import com.onfido.managers.ApplicantManager;
+
+public final class Onfido {
+
+  private static final String DEFAULT_API_URL = "https://api.onfido.com/v3/";
+  private static final String US_API_URL = "https://api.us.onfido.com/v3/";
+
+  public final Config config;
+
+  public final ApplicantManager applicant;
+
+  private Onfido(Builder builder) {
+    config = new Config(builder);
+    applicant = new ApplicantManager(this);
+  }
+
+  public static final class Builder {
+    public String apiToken = "";
+    public String apiUrl = DEFAULT_API_URL;
+
+    private Builder() {}
+
+    public Onfido build() {
+      return new Onfido(this);
+    }
+
+    public Builder apiToken(String apiToken) {
+      this.apiToken = apiToken;
+      return this;
+    }
+
+    public Builder apiUrl(String apiUrl) {
+      this.apiUrl = apiUrl;
+      return this;
+    }
```

Comment:
Could you add a way of setting the region without having to set the url explicitly? e.g.

```java
public Builder regionUS() {
  this.apiUrl = US_API_URL;
  return this;
}
```

Reply:
Done